VideoLocker/src/main/java/org/edx/mobile/view/adapters/CourseDiscussionResponsesAdapter.java

```diff
@@ -219,10 +219,11 @@ private void bindViewHolderToResponseRow(DiscussionResponseViewHolder holder, fi
             @Override
             public void onClick(View v) {
                 if (comment.getChildren().isEmpty()) {
-                    router.showCourseDiscussionAddComment(context, comment);
+                    if (!discussionThread.isClosed())
+                        router.showCourseDiscussionAddComment(context, comment);
```

Comment:
That can be implemented anyway - but we don’t want sound/accessibility reports on clicking non-clickable views ;)
> On 06-Oct-2015, at 7:04 PM, Mi@N Kh@LiD <notifications@github.com> wrote:
> 
> In VideoLocker/src/main/java/org/edx/mobile/view/adapters/CourseDiscussionResponsesAdapter.java <https://github.com/edx/edx-app-android/pull/382#discussion_r41267340>:
> 
> > @@ -219,10 +219,11 @@ private void bindViewHolderToResponseRow(DiscussionResponseViewHolder holder, fi
> >              @Override
> >              public void onClick(View v) {
> >                  if (comment.getChildren().isEmpty()) {
> > -                    router.showCourseDiscussionAddComment(context, comment);
> > +                    if (!discussionThread.isClosed())
> > +                        router.showCourseDiscussionAddComment(context, comment);
> Thats something we actually want in this case, the TalkBack feature will focus on this view and just say that Comments are closed for this response.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/edx/edx-app-android/pull/382/files#r41267340>.
>